src/txkube/_exception.py

```diff
@@ -32,9 +34,19 @@ def __init__(self, code, status):
         self.code = code
         self.status = status
 
-    def __cmp__(self, other):
-        if isinstance(other, self.__class__):
-            return cmp((self.code, self.status), (other.code, other.status))
+    def __lt__(self, other):
+        if isinstance(other, KubernetesError):
+            return (self.code, self.status) < (other.code, other.status)
+        return NotImplemented
+
+    def __eq__(self, other):
+        if isinstance(other, KubernetesError):
+            return (self.code, self.status) == (other.code, other.status)
+        return NotImplemented
+
+    def __ne__(self, other):
+        if isinstance(other, KubernetesError):
+            return (self.code, self.status) != (other.code, other.status)
```

Comment:
I think the `__ne__` is redundant (PiDelport on Freenode suggested this).  [The docs](https://docs.python.org/2/library/functools.html#functools.total_ordering) seem to back this up:

> The class must define one of \_\_lt__(), \_\_le__(), \_\_gt__(), or \_\_ge__(). In addition, the class should supply an \_\_eq__() method.

Reply:
I found that if I removed `__ne__()` here, then I got this failure on Python 2:

```
===============================================================================
[FAIL]
Traceback (most recent call last):
Failure: testtools.testresult.real._StringException: Traceback (most recent call last):
  File "/Users/crodrigues/txkube/src/txkube/test/test_model.py", line 411, in test_from_response
    succeeded(EqualElements()),
  File "/Users/crodrigues/venv-2.7/lib/python2.7/site-packages/testtools/testcase.py", line 498, in assertThat
    raise mismatch_error
testtools.matchers._impl.MismatchError: <KubernetesError: code = 200; status = v1.Status(status=None, code=None, reason=None, details=None, message=None, metadata=None)> != <KubernetesError: code = 200; status = v1.Status(status=None, code=None, reason=None, details=None, message=None, metadata=None)>


txkube.test.test_model.KubernetesErrorTests.test_from_response
-------------------------------------------------------------------------------
Ran 1 tests in 0.004s
```